storage/namespace.go

```diff
@@ -109,10 +120,17 @@ type dbNamespace struct {
 
 	tickWorkers            xsync.WorkerPool
 	tickWorkersConcurrency int
+	statsLastTick          databaseNamespaceStatsLastTick
 
 	metrics databaseNamespaceMetrics
 }
 
+type databaseNamespaceStatsLastTick struct {
+	sync.RWMutex
```

Comment:
This seems like the sort of thing we'd normally use an atomic for, is contention low here?

Reply:
Yes, the only reader will be a background goroutine and the only writer is at the end of a tick (i.e. not in the hot path).